transit_odp/common/utils/aws_common.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import json
+
+from django.conf import settings
+from storages.backends.s3boto3 import S3Boto3Storage
+import boto3
+
+logger = logging.getLogger(__name__)
+
+
+def get_s3_bucket_storage() -> object:
+    """
+    Get AWS S3 bucker storage object
+    """
+    bucket_name = getattr(settings, "AWS_DATASET_MAINTENANCE_STORAGE_BUCKET_NAME", None)
+    if not bucket_name:
+        logger.error("Bucket name is not configured in settings.")
+        raise ValueError("Bucket name is not configured in settings.")
+
+    try:
+        storage = S3Boto3Storage(bucket_name=bucket_name)
+        logger.info(f"Successfully connected to S3 bucket: {bucket_name}")
+        return storage
+    except Exception as e:
+        logger.error(f"Error connecting to S3 bucket {bucket_name}: {str(e)}")
+        raise
+
+
+class SQSClientWrapper:
+    def __init__(self) -> object:
+        """
+        Initialize and return an SQS client.
+        """
+        self.sqs_client = boto3.client(
+            "sqs",
+            endpoint_url=settings.SQS_QUEUE_ENDPOINT_URL,
```

Comment:
Could we use an .env variable to identify local env and set this as the secret keys are not required to connect to sqs in dev and higher envs